src/main/resources/db/migration/V1__.sql

```diff
@@ -0,0 +1,72 @@
+CREATE TABLE category
+(
+    id            INT AUTO_INCREMENT NOT NULL,
+    name          VARCHAR(255)       NOT NULL,
+    `description` VARCHAR(255)       NULL,
```

Comment:
What does the ` do in this context?

Reply:
I guess because of this:
MySQL 8.0 Keywords and Reserved Words:
"DESCRIPTION; added in 8.0.4 (nonreserved)" (https://dev.mysql.com/doc/refman/8.0/en/keywords.html#keywords-8-0-detailed-D)

Reply:
Aha! Neato :)